pkg/applychanges/apply_changes.go

```diff
@@ -65,8 +65,8 @@ func ApplyChanges(changes map[string]interface{}, to interface{}) error {
 				return u, err
 			}
 
-			// If the desination implements graphql.Unmarshaler
-			if reflect.PtrTo(b).Implements(reflect.TypeOf((*graphql.Unmarshaler)(nil)).Elem()) {
+			// If the destination implements graphql.Unmarshaler
+			if reflect.PointerTo(b).Implements(reflect.TypeOf((*graphql.Unmarshaler)(nil)).Elem()) {
```

Comment:
If memory serves, this is just a deprecated function being replaced with a differently named (but functionally equal) one.

Are you able to confirm that was the intent of this change?

Reply:
yes! sorry - should have included in description. updating